.releaserc.js

```diff
@@ -1,7 +1,45 @@
 module.exports = {
   plugins: [
-    '@semantic-release/commit-analyzer',
-    '@semantic-release/release-notes-generator',
+    [
+      '@semantic-release/commit-analyzer',
+      {
+        preset: "angular",
+        releaseRules: [
+          {type: "feat", release: "minor"},
+          {type: "fix", relase: "patch"},
+          {type: "docs", release: false},
+          // but still release if README got changed
+          {type: "docs", scope: "README", release: "patch"},
+          // it still affects the code, but not in any bigger way
+          {type: "style", release: "patch"},
+          {type: "refactor", release: "patch"},
+          {type: "perf", release: "patch"},
+          {type: "test", release: false},
+          {type: "chore", release: false},
+          {type: "dependencies", release: "minor"}
```

Comment:
I agree with `refactor` & `perf ` that they should bump the new version.
---
But for `style` scope it looks quite strange. From [Angular convention](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#type):
> style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)

We are using typescript and most colons, whitespaces will be rewritten on the traspilation/build phase. So publishing a new version without any real changes in the package code looks strange.

---

Also, I'm not sure about bumping a new version for README changes. For example, if somebody changes punctuation or fix a typo, we will publish a new package. I don't feel that this is a very important change and users shouldn't be disturbed via publishing a new version. Most users will be happy to receive the README update with a delay but with additional real fix or improvement in the code. 😉